src/gt4py/next/common.py

```diff
@@ -645,7 +645,7 @@ def _broadcast_ranges(
     _R = TypeVar("_R", _Value, tuple[_Value, ...])
 
     class GTBuiltInFuncDispatcher(Protocol):
-        def __call__(self, func: fbuiltins.BuiltInFunction[_R, _P], /) -> Callable[_P, _R]: ...
+        def __call__(self, /, func: fbuiltins.BuiltInFunction[_R, _P]) -> Callable[_P, _R]: ...
```

Comment:
Why is this change needed? Do we want to use `func` as keyword argument in  dispatchers?

Reply:
as discussed, this makes the protocol match the actual implementation. I think the actual implementation makes sense for this unary function